config/jobs/kubernetes-sigs/cluster-api-provider-digitalocean/cluster-api-provider-digitalocean-periodics.yaml

```diff
@@ -0,0 +1,31 @@
+periodics:
+- name: periodic-cluster-api-provider-digitalocean-conformance-v1alpha3
+  decorate: true
+  decoration_config:
+    timeout: 4h
+  interval: 12h
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+    preset-do-credential: "true"
+  extra_refs:
+    - org: kubernetes-sigs
+      repo: cluster-api-provider-digitalocean
+      base_ref: master
+      path_alias: "sigs.k8s.io/cluster-api-provider-digitalocean"
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20201027-f0200e6-1.18
+      command:
+        - "runner.sh"
+        - "./scripts/ci-conformance.sh"
+      securityContext:
+        privileged: true
+      resources:
+        requests:
+          cpu: 2
+          memory: "9Gi"
```

Comment:
That seems like a lot of memory. What's the reason for requesting that much?

Reply:
I was using that based on the other providers that we have conformance as well

Reply:
Okay, then, let's leave it as it is.